test/config/bignumber_setup.ts

```diff
@@ -0,0 +1,11 @@
+import { BigNumber } from "bignumber.js";
```

Comment:
since we're already including `"@0xproject/utils": "^0.1.0",` can we just use their exported BigNumber? Or I guess we could use our own for more flexibility